package.json

```diff
@@ -29,9 +30,12 @@
     "node": ">= 0.10"
   },
   "dependencies": {
+    "@types/bluebird": "~3.0.36",
+    "@types/express": "~4.0.34",
     "bluebird": "^3.4.0",
     "lodash": "^4.16.0",
-    "validator": "~6.1.0"
+    "typescript": "~2.0.10",
```

Comment:
This should definitely go to `devDependencies` otherwise everyone will get `typescript` installed